corehq/apps/case_importer/templates/case_importer/partials/excel_field_rows.html

```diff
@@ -59,21 +59,28 @@
     <div class="has-warning">
       <p class="help-block">
         {% blocktrans with '<!--ko foreach: caseFieldSuggestions--><!--ko if: $index() !== 0-->, <!--/ko--><strong data-bind="text: $data"></strong><!--/ko-->' as suggestion %}
-          Did you mean {{ suggestion }}?
+          Did you mean "{{ suggestion }}" instead?
         {% endblocktrans %}
       </p>
     </div>
     <!--/ko-->
   </td>
 
-  <td>
+  <td class="col-md-1">
     <div class="checkbox">
       <label>
-        <input type="checkbox" class="new_property" data-bind="checked: isCustom"/>
-        {% trans "Create a new property instead" %}
+        <input type="checkbox" class="new_property" style="margin-left: auto;" data-bind="checked: isCustom"/>
       </label>
     </div>
   </td>
 
+  <td>
+    <!--ko if: valuesHint().length-->
```

Comment:
Style: usually we put the knockout bindings directly on the elements, so the `if` and `foreach` would both go on the `ul`. IIRC, knockout might complain about using `if` and another binding on the same element, but if it does, `visible` would also work fine. We do use the comment syntax, but only when there isn't an appropriate element to attach to.

Reply:
Thanks...I've actually been struggling to discern when which way is preferred...unsuccessfully so far...so hopefully I can make some progress understanding better with this feedback.

Reply:
I would believe there's plenty of inconsistency in the codebase...